### ISSUE TITLE
Navigating to End of Deck slide via ESC menu doesn't exit full screen

### ISSUE THREAD

**Issue by ImperialSquid** [personalizedrefrigerator/joplin-plugin-revealjs-slides#1]
**Describe the bug**
Title

**To Reproduce**
1. Create a slideshow
2. Presnet slide show
3. Press ESC to enter slide naviagation menu
4. Select End of Deck slide

**Expected behavior**
End of Deck slide normally exists full screen and brings up an Exit button

**Platform:**
 - OS: Windows
 - Plugin Version: v0.1.2
 - Joplin Version: v2.11.11

**Additional context**
Navigating away and back with arrow keys resolves the issue so it's just that whatever exits full screen fails to be triggered

**Comment by personalizedrefrigerator** [comment]
It seems that the `slidechanged` event isn't being fired when exiting the overview mode.

There are currently two keyboard shortcuts to open the overview (ESC and O). It might make more sense to remap ESC to showing the "Exit" button (in which case, pressing ESC twice would close the presentation).

**Comment by ImperialSquid** [comment]
Yeah having ESC be quit seems much more natural since that's how PowerPoint does it

**Comment by personalizedrefrigerator** [comment]
ESC has been remapped to toggling the exit button in https://github.com/personalizedrefrigerator/joplin-plugin-revealjs-slides/commit/3e6a609845ada97a917aed7735f0a130da4fd503.

**Comment by ImperialSquid** [comment]
Exc then Enter is fine but is there any reason to not allow Esc anyway?

**Comment by personalizedrefrigerator** [comment]
I was worried that users might accidentally press `Esc` during a presentation and not know how to hide the "Exit" button.

**Comment by ImperialSquid** [comment]
Ah gotcha, so Esc is for early exits but the user using Enter at the end of a presentation is the "intended" path to finishing

**Comment by ImperialSquid** [comment]
I still think pressing Esc to exit a presentation feels more natural, even knowing that Enter was Exit I got a little tripped up testing the new version

Maybe these can just be keybinds left up to the user to decide?

**Comment by personalizedrefrigerator** [comment]
> Maybe these can just be keybinds left up to the user to decide?

For now, I've changed the key that activates "exit" back to (ESC) -- I also found ESC > Enter difficult to use.
